src/base/sysinfo.cc

```diff
@@ -230,15 +334,20 @@ extern "C" {
 // TODO(csilvers): set an envvar instead when we can do it reliably.
 bool GetUniquePathFromEnv(const char* env_name, char* path) {
   char* envval = getenv(env_name);
+  bool is_child;
```

Comment:
it will be more readable to have this in same place as initialization below (and feel free to const it)